client/elements/text/sc-simple-text.html

```diff
@@ -278,6 +288,10 @@ <h3>Could not find the sutta text.</h3>
                       const classrefs = this.$.simple_text_content.querySelectorAll('.'+key);
                       Array.from(classrefs).forEach(item => item.title = this.classTitles[key]);
                   }
+                  for (var key in this.classTitlesTI){
+                      const classrefsTI = this.$.simple_text_content.querySelectorAll('.'+key);
+                      visible ? Array.from(classrefsTI).forEach(item => item.title = this.classTitlesTI[key]) : Array.from(classrefsTI).forEach(item => item.title = "");
```

Comment:
I'd advise keeping complicated logic out of ternary operators (`? :`) - it's just harder to read. I'd rewrite this as:
```
if (visible) {
    Array.from(classrefsTI).forEach(item => item.title = this.classTitlesTI[key])
} else {
    Array.from(classrefsTI).forEach(item => item.title = "");
}
```

[Here's](https://github.com/airbnb/javascript#comparison--nested-ternaries) the relevant part of the Airbnb JS style guide.

Reply:
OK. Good. Will change that. I actually had that at first but changed it.

Reply:
Please use single quotes instead of double quotes.